lib/python/isodatetime/main.py

```diff
@@ -285,7 +285,10 @@ def main():
         ],
     ]:
         arg_parser.add_argument(*o_args, **o_kwargs)
-    args = arg_parser.parse_args()
+    if hasattr(arg_parser, 'parse_intermixed_args'):
+        args = arg_parser.parse_intermixed_args()
+    else:
+        args = arg_parser.parse_args()
```

Comment:
Remember to raise a PR on `isodatetime` for these 

- chages.

Reply:
I thought I had. [isodatetime PR_#124](https://github.com/metomi/isodatetime/pull/124)